x-pack/legacy/plugins/fleet/dev_docs/api/agents_unenroll.md

```diff
@@ -0,0 +1,38 @@
+# Enroll Fleet agent API
+
+Unenroll an agent
+
+## Request
+
+`POST /api/fleet/agents/unenroll`
+
+## Request body
+
+- `ids` (Optional, string) An list of agent id to unenroll.
+- `kuery` (Optional, string) a kibana query to search for agent to unenroll.
```

Comment:
we should add a note that only one of these params is allowed (can't specify both `ids` and `kuery` at the same time)